src/utils/constants/events.js

```diff
@@ -1,4 +1,13 @@
 export const pastEventData = [
+  {
+    imageBackgroundColor: null,
+    date: 'April 13, 2021',
```

Comment:
heads up the previous events just use <month> <year>

might be worth matching for consistency :)

Reply:
Sorry, I do not understand.

Did you mean the date format?

Reply:
oh whoops, sorry github treated them as HTML, edited :)

Yes I'm suggesting `April 2021`
```suggestion
    date: 'April 2021',
```

Reply:
Ah, last month when Joe and I updated it, we decided to start including the actual day as well.  I guess we should have put something about changing the format.

Reply:
ah ok, nevermind then carry on! :)

Reply:
Thanks!